From ec5280ed2d252565bb845500398ccf72990b5c7e Mon Sep 17 00:00:00 2001
In-Reply-To: <xmqq7ez6hk2y.fsf@gitster.mtv.corp.google.com>
References: <xmqq7ez6hk2y.fsf@gitster.mtv.corp.google.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 20 Jul 2017 14:11:42 +0000
Subject: [PATCH 4/6] RelNotes: mention that PCRE v2 exposes the same syntax
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

For someone not familiar with PCRE or having read its own
documentation this isn't obvious, let's explicitly mention it so
package maintainers won't fear upgrading least they break things for
their users.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/RelNotes/2.14.0.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/RelNotes/2.14.0.txt b/Documentation/RelNotes/2.14.0.txt
index 7ed93bca37..0e363f2af3 100644
--- a/Documentation/RelNotes/2.14.0.txt
+++ b/Documentation/RelNotes/2.14.0.txt
@@ -28,6 +28,9 @@ Backward compatibility notes and other notable changes.
    upstream PCRE maintainer has abandoned v1 maintenance for all but
    the most critical bug fixes, use of v2 is recommended.
 
+   Version v2 of the library is fully backwards compatible with the
+   Perl-compatible regular expression syntax exposed by git (sans a
+   few obscure bugfixes).
 
 Updates since v2.13
 -------------------
-- 
2.13.2.932.g7449e964c

